### ISSUE TITLE
File cache conflict

### ISSUE THREAD

**Issue by Jasperav** [intellij-rust/intellij-rust#9289]
## Environment

* **IntelliJ Rust plugin version: 0.4.177.4844-222**
* **Rust toolchain version: 1.63**
* **IDE name and version: CLion 2022.2**
* **Operating system: MacOS**

## Problem description
This windows pop up very frequently the last days: 

![image](https://user-images.githubusercontent.com/20363565/188195510-18ffc82e-cb55-4d87-8c9c-5b2731cdceac.png)

## Steps to reproduce

1. Write some rust code
2. Run your rust code with a hotkey

**Comment by vlad20012** [comment]
It's weird. Perhaps you have something non-usual in your environment, or in IDE configuration, or in how you build the code? Maybe there is some tool that reformat the code when you save it?

**Comment by Jasperav** [comment]
@vlad20012 No, none of that. I double checked in my settings the Rustfmt does not run after every save :(
